parlai/mturk/core/server/html/core.html

```diff
@@ -523,10 +525,20 @@ <h1>Live Chat</h1>
       // Heartbeats ensure we're not disconnected from the server
       log('received heartbeat: ' + msg.id, 4);
       pongs_without_heartbeat = 0;
+      heartbeats_without_pong += 1;
```

Comment:
Ah I see the confusion here - heartbeats without pong should refer to _sent_ heartbeats without receiving a pong back.